docs/pages/includes/aws-no-credential-provider.mdx

```diff
@@ -0,0 +1,17 @@
+### No credential providers error
+
+If you see the error `NoCredentialProviders: no valid providers in chain` in {{service}} Service logs then Teleport
+is not detecting the required credentials to connect via AWS IAM permissions. Check whether
+the credentials or security role has been applied in the machine running the Teleport {{service}} Service.
+
+When running on EKS, this error may occur if the Teleport {{service}} Service
+cannot access IMDSv2 when PUT requests hop limit on the worker node instance is
+set to 1. You can check the value of `HttpPutResponseHopLimit` by running
+command [`aws ec2
```

Comment:
I would put example commands on their own lines using code fences to make this easier to read.